html-web-page/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HTML Web Page</title>
+  <style>
+    table,
+    th,
+    td {
+      border: 1px solid black;
+    }
+
+    td:not(:first-child) {
+      text-align: center;
+    }
+  </style>
+</head>
+<body>
+
+  <!-- your HTML code here 👇 -->
+<h1>Jelly of the Month Club</h1>
+<h3>Make all your friends jelly of your of your stock of jelly!</h3>
+<div> Here at JOTMC Inc., we are constantly striving to supply our customers with the highest-quality and most unique jellies on a monthly basis. For the jelly love in your
+  life (or yourself!), this is the figt that keeps on giving all year round. We offer three tiers of monthly jelly joy, and each one offers their own perks. Here is a breakdown
+  of each, as well as your monthly price:
+</div>
+<p><!-- Entered for the purpose of the line break.!--></p>
```

Comment:
You definitely don't want to use `p` elements as a line break. What you can do is put the text above into a `p` element instead of a `div` element! The `div` element is generally meant to be used to break off a section of the document for styling and organization, it shouldn't be used as a text element. The `p` element is the ideal element for general text without any specific styling

Reply:
Understood.